README.md

```diff
@@ -27,36 +27,36 @@ These PLC variants are supported:
 * Mitsubishi iQR series
 * Mitsubishi iQF series
 
-The SLMP Connector is an application that runs on the individual IED. Connections can be configured using the IIH Configurator for Industrial Edge. The connector transfers the value series of selected data points from a PLC to the Databus. From there the data can be used within other Edge apps, e.g. the Flow Creator.
+The SLMP Connector is an application that runs on the individual Industrial Edge Device. Connections can be configured using the Common Configurator for Industrial Edge. The connector transfers the value series of selected data points from a PLC to the Databus. From there the data can be used within other Edge apps, e.g. the Flow Creator.
 
 ### General task
 
-Here we configure a connection to a Mitsubishi PLC FX5U (iQF) using the SLMP Connector. The FX5U acts as SLMP server and the IED acts as SLMP client. The data is published on the IE Databus. By using the application IE Flow Creator, we fetch the metadata of the SLMP Connector, write some data on the configured tags and read out the new data.
+Here we configure a connection to a Mitsubishi PLC FX5U (iQF) using the SLMP Connector. The FX5U acts as SLMP server and the Industrial Edge Device acts as SLMP client. The data is published on the Databus. By using the application Flow Creator, we fetch the metadata of the SLMP Connector, write some data on the configured tags and read out the new data.
 
-![overview](/docs/graphics/Overview.png)
+![overview](/docs/graphics/Slmparchi.png)
 
 ## Requirements
 
 ### Prerequisites
 
 * Access to an Industrial Edge Management (IEM) with onboarded Industrial Edge Device (IED)
 * IEM: Installed system configurator for Databus
-* IED: Installed apps IE Databus, IE Flow Creator, IIH Configurator, IIH Registry Service, SLMP Connector, Data Service (optional)
+* IED: Installed apps Databus, Flow Creator, common Configurator, Registry Service, SLMP Connector, IIH Essentials (optional)
```

Comment:
Please write "Common" instead of "common"

Reply:
I changed this